subcommands/status/cmd.go

```diff
@@ -87,8 +87,8 @@ func printTargetStatus(tagPrefix string, tagStatus []client.TagStatus) {
 		}
 		fmt.Printf("\n## %s Tag: %s\n", tagPrefix, name)
 		// Tabby doesn't indent (or at least easily) so:
-		fmt.Println("\tTARGET   DEVICES  DETAILS")
-		fmt.Println("\t-------  -------  -------")
+		fmt.Println("\tTARGET   DEVICES  INSTALLING  DETAILS")
```

Comment:
Should this be "re-installing"?

Reply:
The most proper form is `(re-)installing`, because that includes both `installing` and `reinstalling` (at least it seems to me a server side logic does exactly that).
But that word simply seems too long... IMHO installing is enough informative.